tasks/watch.cr

```diff
@@ -262,7 +264,14 @@ module LuckySentry
       if build_success
         self.app_built = true
         create_app_processes
-        puts "#{" Done ".colorize.on_cyan.black} compiling"
+
+        elapsed_time = Time.monotonic - @build_started
+        message = String.build do |io|
+          io << " DONE ".colorize.on_cyan.black
+          io << " Compiled successfully in #{elapsed_time}".colorize.cyan
```

Comment:
What about calling distance_of_time_in_words or something on this so it's a little easier on the eyes?

Reply:
oh, that's a good idea. I was thinking I didn't want to write out a whole deal, but I forgot I could just use that.

Reply:
Alright, made these changes and updated the screenshot. It looks a lot better.

Reply:
excellent!